### ISSUE TITLE
[Adaptive Card iOS - ChoiceSet.DynamicTypeahead.json]: Screen Reader is not conveying the search suggestion information along with total number of records under Enter the name of your field.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#172]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.6" control and select it.
4. Navigate to 'Elements' control and activate it.
5. Navigate to 'ChoiceSet.DynamicTypeahead.json' under 'Elements' list item and activate it.
6. Card will be appear.
7. Perform right swipe, navigate to the Adaptive Cards button, and double tap to activate it.
8. Enter the name of your team screen will be displayed.
9. Perform right swipe and nvaigate to the edit field and type some chatacter and observe the issue.

**Actual Result:**
After entering the character under the Adaptive Cards edit field, screen reader does not announce the information about the matching records along with the total number of search results. The screen reader remains silent.

**Expected Result:**
Screen Reader should announce the information about the matching records along with total count of the matching result.

**Note:**
Same issue is repro on V1.6 > Test >Input.ChoiceSet.Dynamic.Typeahead.Json.

**User Impact:**
It will impact the screen reader user as they will not be able to know the total number of the search result as well as they will not be able to know matching records due to this, they will not be able to know whether search results are getting displayed or not.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/858cbefa-e7d8-49a3-a306-cc4a66e21bac

### Card JSON

```JSON
NA
```

**Comment by MengliElmendorf137** [comment]
I think this is expected behavior. The voice announce the search results when user navigates to the search result list. Attached recording for reference.https://github.com/user-attachments/assets/14594d6d-672e-442d-af11-142c57eeedcb
